raiden/utils/packing.py

```diff
@@ -75,3 +83,30 @@ def pack_reward_proof(
             nonce,
         ],
     )
+
+
+def pack_withdraw(
+    canonical_identifier: CanonicalIdentifier, participant: Address, total_withdraw: WithdrawAmount
+) -> bytes:
+    """Packs balance proof data to be signed
+
+    Packs the given arguments in a byte array in the same configuration the
+    contracts expect the signed data to have.
+address(this),
```

Comment:
the doc string format specification looks weird, why `address(this)`?
the format below looks more like
```
token_network_address,
chain_id,
uint256(MessageTypeId.Withdraw),
channel_identifier,
participant_address,
total_withdraw
```